### DISCUSSION TITLE
Closing vorta from GUI does not terminate running `vorta` command in terminal

### DISCUSSION THREAD

**Discussion by DaffyTheDuck** [borgbase/vorta#1581] [General]
When you run the `vorta` command from the terminal to launch new session and close the application from GUI instead of closing it from the terminal, the command `vorta` keeps running though the application is closed

![image](https://user-images.githubusercontent.com/34976093/216822026-12e3a800-5176-4538-a33e-4261f808cfb4.png)

## How to reproduce ?
1. Launch vorta from terminal
2. Close it through GUI
3. See the terminal

## Desktop
Ubuntu 22.04

## Update!
I actually just found that it keeps running in background! But while closing the application it would be nice to prompt user whether he want's to continue the process or not 😄

**Comment by real-yfprojects** [comment]
See #1582.
